x-pack/legacy/plugins/snapshot_restore/__jest__/client_integration/policy_add.test.ts

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { act } from 'react-dom/test-utils';
+
+import * as fixtures from '../../test/fixtures';
+
+import { setupEnvironment, pageHelpers, nextTick, getRandomString } from './helpers';
+import { PolicyFormTestBed } from './helpers/policy_form.helpers';
+import { DEFAULT_POLICY_SCHEDULE } from '../../public/app/constants';
+
+const { setup } = pageHelpers.policyAdd;
+
+jest.mock('ui/i18n', () => {
+  const I18nContext = ({ children }: any) => children;
+  return { I18nContext };
+});
+
+const POLICY_NAME = 'my_policy';
+const SNAPSHOT_NAME = 'my_snapshot';
+const MIN_COUNT = '5';
+const MAX_COUNT = '10';
+const EXPIRE_AFTER_VALUE = '30';
+const repository = fixtures.getRepository({ name: `a${getRandomString()}`, type: 'fs' });
+
+// We need to skip the tests until react 16.9.0 is released
+// which supports asynchronous code inside act()
+describe.skip('<PolicyAdd />', () => {
```

Comment:
Just to clarify, none of the tests are currently running in CI - are we manually testing for now?

Reply:
that's correct. we can remove the `skip` once we upgrade to react `16.9.0`.